content/en/docs/genai/_index.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "GenAI"
```

Comment:
Also, shall we call the title also GenAI with Kubeflow or you prefer shorter titles?

Reply:
because you get this:

![Screenshot 2025-06-04 at 12 23 59 PM](https://github.com/user-attachments/assets/a38bf3e6-5965-4eef-acb6-d4d22c771944)